### ISSUE TITLE
ONNX export

### ISSUE THREAD

**Issue by jsvir** [nvidia-riva/riva-asrlib-decoder#1]
Is it possible to export it to onnx? Are there other ways to run it on triton server?

**Comment by messiaen** [comment]
Triton does support onnx.  This library performs wfst decoding directly on a ctc output logits, which can be generated however you want.

[Riva](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-overview.html) supports ONNX models via [NeMo](https://github.com/NVIDIA/NeMo).
